Homework1_EliseevAN/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+<header class="menu">
+    <button id = "1" class="cart-button" type="button">Корзина</button>
+</header>
+    <div class="catalog"></div>
+<script>
+
+    alert('Нажмите на кнопку "Корзина"');
+    const goods = [
+        {},
+        { title: 'Жесткий диск SSD 1Tb', price: 10000 },
+        { title: 'Материнская плата', price: 4000 },
+        { title: 'Видеокарта', price: 15000 },
+    ];
+    // Cобытие для кнопки
+    let button = document.getElementById("1");
```

Comment:
Рекомендую переменные, в которых хранятся DOM-узлы, давать имена, начинающиеся с $, чтобы можно было легко отличить их от обычных переменных. Так же обратите внимание на оформление кода, нужно постараться использовать только один вид кавычек
```suggestion
    const $button = document.getElementById('1');
```